src/index.ts

```diff
@@ -684,3 +712,7 @@ function getWebsiteUrl(hostName: string): string {
     const domain = getDomain();
     return `https://${hostName}${domain}`;
 }
+
+function getSubId(input: string): string {
```

Comment:
don't get what this is for. It is returning the same value that it gets as parameter